onnxruntime/contrib_ops/cuda/quantization/qordered_ops/qordered_attention.cc

```diff
@@ -75,11 +75,11 @@ Status QOrderedAttention::PutIntoMergedWeightScale(const Tensor& tensor, Allocat
   auto offset = std::accumulate(&qkv_hidden_sizes_[0], &qkv_hidden_sizes_[qkv_index], 0LL);
   float* target = ((float*)merged_qkv_alpha_.get()) + offset;
   int count = gsl::narrow_cast<int>(qkv_hidden_sizes_[qkv_index]);
-  CUBLAS_RETURN_IF_ERROR(cublasScopy(CublasHandle(), count, tensor.Data<float>(), tensor.Shape().IsScalar() ? 0 : 1, target, 1));
+  CUBLAS_RETURN_IF_ERROR(cublasScopy(DefaultCublasHandle(), count, tensor.Data<float>(), tensor.Shape().IsScalar() ? 0 : 1, target, 1));
   ORT_ENFORCE(const_scale_input_ > 0.0f && const_scale_qkv_layer_[qkv_index] > 0.0f,
               "input scale and respective qkv gemm scale must be positive constant!");
   float scale = static_cast<float>((double)const_scale_input_ / const_scale_qkv_layer_[qkv_index]);
-  CUBLAS_RETURN_IF_ERROR(cublasSscal(CublasHandle(), count, &scale, target, 1));
+  CUBLAS_RETURN_IF_ERROR(cublasSscal(DefaultCublasHandle(), count, &scale, target, 1));
```

Comment:
use GetCublasHandle(context) instead?

Reply:
this is during initialization phase, context is not there yet.